test/factory.spec.ts

```diff
@@ -488,6 +490,58 @@ describe('Factory', () => {
       expect(address).toMatchObject(addressAttributes);
       expect(address.userId).toBe(userAttributes.id);
     });
+
+    it('it should not create overridden associations', async () => {
```

Comment:
This feature is something that is not exclusive to Sequelize, so it'd be better to test it with the Object Adapter. If you want to double check this is not creating a new entry in the database when used with Sequelize, I advise also adding a test to the [sequelize-specific test suite](https://github.com/thiagomini/factory-girl-ts/blob/cef853f65206e892437eb10c1331fb3e48ae3a53/test/integration/sequelize.e2e-spec.ts)

Reply:
@thiagomini sorry I didn't notice this for a while - I've changed it to `ObjectAdapter` 👍  doesn't seem like the issue was specific to sequelize so no need to put it into a specific test suite I think